.eslintrc.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
+        node: true,
+    },
+    settings: {
+        "import/resolver": {
+            node: {
+                extensions: [".js", ".jsx", ".ts", ".tsx"],
+                moduleDirectory: ["node_modules", "src/"],
+            },
+        },
+        react: {
+            version: "detect",
+        },
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:prettier/recommended",
+        "plugin:import/recommended",
+    ],
+    globals: {
+        Atomics: "readonly",
+        SharedArrayBuffer: "readonly",
+    },
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true,
+        },
+        ecmaVersion: 11,
+        sourceType: "module",
+    },
+    plugins: ["react", "@typescript-eslint", "import"],
+    rules: {
+        "no-unused-var": "off",
+        "no-var": "error",
+        curly: ["error", "multi-line"],
+        "import/order": [
+            "error",
+            { groups: ["index", "sibling", "parent", "internal", "external", "builtin", "object", "type"] },
+        ],

```

Comment:
```suggestion
        ],
        "prettier/prettier": [
            "error", {
                "endOfLine": "auto"
            }
        ],
```
Add rule to eslint file to avoid lint error `[eslint] Delete CR [prettier/prettier]` for Windows users.